realm/realm-library/src/main/java/io/realm/internal/RowObject.java

```diff
@@ -0,0 +1,117 @@
+package io.realm.internal;
+
+import io.realm.ObjectChangeSet;
+import io.realm.RealmObjectChangeListener;
+
+/**
+ * Java wrapper for Object Store's {@code Object} class. Currently it is only used for object notifications.
+ */
+public class RowObject implements NativeObject {
```

Comment:
I am really out of ideas ... First i tried `Object`, but apparently it will conflict a lot with java's `Object` ... But `ObjectStoreObject` sounds really stupid... any suggestions are welcome.